dovpanda/core.py

```diff
@@ -114,6 +114,18 @@ def check_csv_size(arguments):
                     f'try:  <code>pd.read_csv({filename}, nrows=5)</code> to check schema is as expected.')
 
 
+@ledger.add_hint(config.WRITE_TEXT_METHODS, 'post')
+def suggest_zipping_on_to_csv(res, arguments):
+    filename = arguments.get('path_or_buf')
+    if not os.path.exists(filename):
+        return
+    if os.path.getsize(filename) > config.MAX_CSV_SIZE:
+        source_func = arguments.get(config.DOVPANDA_INFO_DICT)[config.SOURCE_FUNC_NAME]
+        ledger.tell('Saved file size is very large. '
+                    'If you would like to save some space, try zipping on the fly by using the compression keyword, '
+                    f'try:  <code>pd.{source_func}({filename}, compression=\'gzip\')</code>')
```

Comment:
put code part in separate line

Reply:
(y)

Reply:
disclaimer: different line for the user - meaning a `<br>` tag